package.json

```diff
@@ -4,6 +4,8 @@
   "description": "A lightweight ORM for PostgreSQL, MySQL, and SQLite3",
   "main": "bookshelf.js",
   "scripts": {
+    "precommit": "lint-staged",
```

Comment:
Is there a reason to use the `lint-staged` approach over `pretty-quick` as described here https://prettier.io/docs/en/precommit.html ?

Reply:
The main reason is that `knex` uses `lint-staged`.

And some reference:

- lint-staged 
npm weekly downloads: 479,223
github stars: 3140

- pretty-quick
npm weekly downloads: 54,925
github stars: 386

Reply:
We can go with this approach for now. If anyone has any reasons to use `pretty-quick` we can discuss this further.